docs/cloudformation_compatibility.rst

```diff
@@ -184,6 +184,7 @@ MinimumCompressionSize              All
 Cors                                All
 TracingEnabled                      All
 OpenApiVersion                      None
+Domain                              All
```

Comment:
Are you sure this is true?

Reply:
I added the intrinsic support now and added a test for it.